### PR TITLE
Cover Element should cover all matching elements

### PR DESCRIPTION
**Pull request by SergiuTudos**
For specific element we'll need to use specific locator

### CODE REVIEW

File: atest/resources/html/links.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
       <a href="http://www.google.com">Absolute external link</a><br/>
       <a id="some_id" href="broken.html">Link with id</a><br/>
       <a href="index.html"><img src="image.jpg" id="image_id"/></a><br/>
+      <a href="index.html"><img src="image.jpg" id="image_id"/></a><br/>
```
Comment (aaltat) [inline atest/resources/html/links.html]: Having same id is not allowed by some browsers. Please change the id.

Comment (SergiuTudos) [reply atest/resources/html/links.html]: Updated and ready to merge

**Comment by aaltat** [comment]
I have few question
1) If element is not found keyword does not fail, but should it fail? I have leaning towards that it should fail, but I do not feel strongly on the matter. But regardless which way the keyword will work, situation must have a test. 
2) Should keyword return count of how many elements it did cover? Feel that it could useful feature to check that correct elements are covered.

**Comment by SergiuTudos** [comment]
I added the case when element is not found
For returning elements count I think we're going to do over-engineering to the keyword. Who needs this can use get elements count.

**Comment by SergiuTudos** [comment]
can someone rerun tests?

**Comment by HelioGuilherme66** [comment]
@SergiuTudos 
Done.
Green again!

**Comment by SergiuTudos** [comment]
Thank you !!!
Waiting for sensei to merge code

**Comment by aaltat** [comment]
Fixes #1239

**Pull request merged**